.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,25 @@
+# Sync GitHub issues to Jira issues
+
+# Configuration syntax:
+# https://github.com/canonical/gh-jira-sync-bot/blob/main/README.md#client-side-configuration
+settings:
+  # Repository specific settings
+  components: # Jira components that will be added to Jira issue
+    - postgresql-vm
+    - postgresql-k8s
+    - pgbouncer-vm
+    - pgbouncer-k8s
```

Comment:
@taurus-forever is pgbouncer still relevant for the postgresql snap now that it has its own?